eZ/Publish/Core/Persistence/Legacy/Content/Location/Trash/Handler.php

```diff
@@ -109,6 +109,7 @@ public function trashSubtree($locationId)
         $locationRows = $this->locationGateway->getSubtreeContent($locationId);
         $isLocationRemoved = false;
         $parentLocationId = null;
+        $trashedLocationsContentMap = [];
```

Comment:
let's be consistent:
```suggestion
        $removedLocationsContentMap = [];
```
(or the other way, other variables starting with trashed instead).